packages/ignite/src/templates/plugin/templates/Example.js.ejs

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import React from 'react'
+import { View, Text } from 'react-native'
+import ExamplesRegistry from '../../../App/Services/ExamplesRegistry'
```

Comment:
👍  as long as it doesn't break things.  We should try pushing a production build with it to ensure there's no side effects.  It seems great.